src/encoder.rs

```diff
@@ -1935,9 +1935,9 @@ fn write_tile_group_header(tile_start_and_end_present_flag: bool) ->
 
 pub fn encode_frame(sequence: &mut Sequence, fi: &mut FrameInvariants, fs: &mut FrameState) -> Vec<u8> {
     let mut packet = Vec::new();
-    //write_uncompressed_header(&mut packet, sequence, fi).unwrap();
-    write_obus(&mut packet, sequence, fi, fs).unwrap();
     if fi.show_existing_frame {
+        //write_uncompressed_header(&mut packet, sequence, fi).unwrap();
```

Comment:
Because you're touching this code anyway, might as well drop the commented out cruft.